Tools/cases_generator/tier1_generator.py

```diff
@@ -41,20 +41,35 @@ def declare_variables(inst: Instruction, out: CWriter) -> None:
         if isinstance(uop, Uop):
             for var in reversed(uop.stack.inputs):
                 if var.name not in variables:
-                    type = var.type if var.type else "PyObject *"
                     variables.add(var.name)
+                    type = var.type if var.type else "PyStackRef "
                     if var.condition:
-                        out.emit(f"{type}{var.name} = NULL;\n")
+                        if type.strip() == "PyStackRef":
+                            out.emit(f"_PyStackRef {var.name} = Py_STACKREF_NULL;\n")
+                        else:
+                            out.emit(f"{type}{var.name} = NULL;\n")
                     else:
-                        out.emit(f"{type}{var.name};\n")
+                        if var.is_array():
+                            out.emit(f"{var.type}{var.name};\n")
+                        else:
+                            if type.strip() == "PyStackRef":
```

Comment:
Why are we using "PyStackRef" for the typename here, and `_PyStackRef` in the C code?

Reply:
It was a bug. Fixed.